index.js

```diff
@@ -29,6 +31,12 @@ function extract(mbTilesPath, geojson, propName) {
         var writeQueue = {};
         var writable = {};
 
+        db.getInfo(function (err, info) {
+            if (err) throw err;
+
+            meta = info;
+        });
```

Comment:
For small jobs, this may happen after we processed the tiles. It's better to add the callback to the final queue to avoid any race conditions.